android/app/src/main/java/org/prideinlondon/festival/MainApplication.java

```diff
@@ -4,6 +4,13 @@
 
 import com.crashlytics.android.Crashlytics;
 import com.facebook.react.ReactApplication;
+import org.devio.rn.splashscreen.SplashScreenReactPackage;
```

Comment:
Another side effect of RN link. All these additions can be reverted

Reply:
👍